admin_guide/manage_nodes.adoc

```diff
@@ -315,67 +315,67 @@ xref:marking-nodes-as-unschedulable-or-schedulable[marked unschedulable] to
 
 Only pods backed by a
-xref:../architecture/core_concepts/deployments.adoc#replication-controllers[replication
-controller] can be evacuated; the replication controllers create new pods on
-other nodes and remove the existing pods from the specified node(s). Bare pods,
-meaning those not backed by a replication controller, are unaffected by default.
-You can evacuate a subset of pods by specifying a pod-selector. Pod selector is                                                                                             
+xref:../architecture/core_concepts/deployments.adoc#replication-controllers[replication controller]
+can be evacuated; the replication controllers create new pods on other nodes and
+remove the existing pods from the specified node(s). Bare pods, meaning those
+not backed by a replication controller, are unaffected by default. You can
+evacuate a subset of pods by specifying a pod-selector. Pod selector is
 based on labels, so all the pods with the specified label will be evacuated.
 
-To evacuate all or selected pods on one or more nodes:
+To evacuate all or selected pods on a node:
 
 ----
-$ oc adm drain <node1> <node2> [--pod-selector=<pod_selector>]
+$ oc adm drain <node> [--pod-selector=<pod_selector>]
 ----
 
 You can force deletion of bare pods by using the `--force` option. When set to
 `true`, deletion continues even if there are pods not managed by a replication
 controller, ReplicaSet, job, daemonset, or StatefulSet:
 
 ----
-$ oc adm drain <node1> <node2> --force=true
+$ oc adm drain <node> --force=true
 ----
 
 You can use `--grace-period` to set a period of time in seconds for each pod to
-terminate gracefully. If negative, the default value specified in the pod will
-be used:
+terminate gracefully. If negative, the default value specified in the pod is
+used:
 
 ----
-$ oc adm drain <node1> <node2> --grace-period=-1
+$ oc adm drain <node> --grace-period=-1
 ----
 
 You can use `--ignore-daemonsets` and set it to `true` to ignore
 daemonset-managed pods:
 
 ----
-$ oc adm drain <node1> <node2> --ignore-daemonsets=true
+$ oc adm drain <node> --ignore-daemonsets=true
 ----
 
 You can use `--timeout` to set the length of time to wait before giving up. A
 value of `0` sets an infinite length of time:
 
 ----
-$ oc adm drain <node1> <node2> --timeout=5s
+$ oc adm drain <node> --timeout=5s
 ----
 
 You can use `--delete-local-data` and set it to `true` to continue deletion even
-if there are pods using emptyDir (local data that will be deleted when the node
-is drained):
+if there are pods using emptyDir (local data that is deleted when the node is
+drained):
 
 ----
-$ oc adm drain <node1> <node2> --delete-local-data=true
+$ oc adm drain <node> --delete-local-data=true
 ----
 
 To list objects that will be migrated without actually performing the evacuation,
 use the `--dry-run` option and set it to `true`:
 
 ----
-$ oc adm drain <node1> <node2>  --dry-run=true
+$ oc adm drain <node>  --dry-run=true
```

Comment:
Super minor but there's an extra space here in case you care to remove.

Reply:
Oh I care greatly. Thanks!